### ISSUE TITLE
Embedding a Table

### ISSUE THREAD

**Issue by IdaraNabuk** [sanette/bogue#36]
Hi Sanette, I am still working on the GUIs to ensure it's complete and can be added to the Bogue repo. I am having issues regarding the usage of a table. 

I want to use a table in the CRUD GUI to store the names, so I can click on a particular name and operate on it. 
I also want to make the table I used in the cell GUI responsive to input. 

What should I do at this point? Here's the link to the [CRUD](https://github.com/IdaraNabuk/Bogue-examples/tree/main/crud) and [Cells](https://github.com/IdaraNabuk/Bogue-examples/tree/main/cells) GUI code base.

**Comment by sanette** [comment]
> I am having issues regarding the usage of a table.

yes, the Table layout was designed to select entire rows, not single cells. 
(well, maybe by selecting the tiny space between rows you can do it but it's clearly not a good solution)
Could you open a separate issue for this? (something like "allow Table to select individual cells")

In the mean time you should not use a Table for your spreadsheet, but rather a custom array of Text_inputs
